transformer_engine/pytorch/module.py

```diff
@@ -993,6 +1031,11 @@ class LayerNormLinear(TransformerEngineBaseModule):
                              together with the output of the linear transformation.
                              Example use case: residual connection for transformer module is
                              taken post layernorm.
+    wandb_param_split : Tuple[str, ...], default = None
```

Comment:
What is the reason for that name?

Reply:
Just to make clear that it is for "weights and biases parameter split"

Reply:
Hmmm, if you did not tell me that I would think that we have some dependency on WandB :-P. Could we make it a different name?

Reply:
Makes sense